docs/resources/User.md

```diff
@@ -227,7 +227,7 @@ Create a new DM channel with a user. Returns a [DM channel](#DOCS_RESOURCES_CHAN
 
 ## Create Group DM % POST /users/@me/channels
 
-Create a new group DM channel with multiple users. Returns a [DM channel](#DOCS_RESOURCES_CHANNEL/channel-object) object. This endpoint was intended to be used with the now-deprecated GameBridge SDK. DMs created with this endpoint will not be shown in the Discord client. Fires a [Channel Create](#DOCS_TOPICS_GATEWAY_EVENTS/channel-create) Gateway event.
+Create a new group DM channel with multiple users. Returns a [DM channel](#DOCS_RESOURCES_CHANNEL/channel-object) object. This endpoint was intended to be used with the now-deprecated GameBridge SDK. Fires a [Channel Create](#DOCS_TOPICS_GATEWAY_EVENTS/channel-create) Gateway event.
```

Comment:
does the bot get actually a channel create event?

Reply:
Yes. Example payload:
```json
{
    "id": "channel_id_here",
    "type": 3,
    "last_message_id": null,
    "flags": 0,
    "managed": true,
    "application_id": "application_id_here",
    "recipients": [
        {
            "id": "user_id_here",
            "username": "username_here",
            "avatar": null,
            "avatar_decoration": null,
            "discriminator": "0001",
            "public_flags": 0
        }
    ],
    "name": "app_name_here",
    "icon": null,
    "owner_id": "app_id_here"
}
```